lib/lbt/bundle/Builder.js

```diff
@@ -42,7 +43,7 @@ const UI5BundleFormat = {
 		outW.write(`jQuery.sap.registerPreloadedModules(`);
 		outW.writeln(`{`);
 		if ( section.name ) {
-			outW.writeln(`"name":"${section.getSectionName()}",`);
+			outW.writeln(`"name":"${section.name}",`);
```

Comment:
Sorry, you might have to rebase again. The same fix has been made in PR #341 where I also tried to close some code coverage gaps un the builder and resolver.

Reply:
🙈